### PR TITLE
IRMG Vaquero-class Light Frigate

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request
vaquero
vaquero_areas

Adds the Vaquero-class Light Frigate, a small (7 players) Inteq generalist to fill the frankly terrible shoes the Hound left behind. While the Vaquero has all the features expected of an Inteq ship (meeting room, armory, large cargo bay, boombox), it actually has average engineering gear in combination with a full set of surgical tools and table, making it much more well-rounded than the other Inteq ships, even if it pays for this by only being moderately armed and having a small footprint and crew capacity.

The Vaquero has a crew capacity of 7:
- one Vanguard
- one Master-At-Arms to run the armory and function as second-in-command
- one Enforcer
- one Artificer
- one Corpsman
- two Recruits

## Why It's Good For The Game

A generalist rounds out the Inteq catalogue nicely, and helps balance out the fact that all of the Inteq specialists are positively enormous.

## Changelog

:cl:
add: Added the IRMG Vaquero
/:cl:

### CODE REVIEW

File: _maps/configs/inteq_vaquero.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"prefix": "IRMV",
+	"description": "The Vaquero is a flexible and cheap military conversion of a light commercial freighter, widely used by Inteq Risk Management Group for picket, reconnaissance, and light escort duty, or for contracts that don't warrant a larger, more specialized ship. Unlike most Inteq ships, the Vaquero features basic medical, engineering, and military equipment all in one package, and its minimal crew requirements are an excellent fit for Vanguards with a smaller command and for training missions. Like its peers, it features a large cargo bay, which can be reconfigured as needed.",
+	"tags": [
+		"Generalist"
+	],
+	"namelists": [
+		"MYTHOLOGICAL",
+		"BEASTS",
+		"NATURAL_AGGRESSIVE",
+		"INTEQ"
```
Comment (Erikafox) [inline _maps/configs/inteq_vaquero.json]:
```suggestion
		"BEASTS",
		"INTEQ"
```

File: _maps/configs/inteq_vaquero.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Vaquero-Class Light Frigate",
```
Comment (Erikafox) [inline _maps/configs/inteq_vaquero.json:3]: not sold on the name honestly. Very Solar flavoring. The lorbs might have some animal (perhaps a kalixian draft beast) that fits better.

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
I am firmly against terminals over catwalks.
dreamseeker_GDIU0mylTM
speaking of, the fuel crate needs to be subtyped, not var-edited.
image
the SSU is eating the intercom. This is an intercom as items problem, but. For now just move the intercom.
dreamseeker_fCQlaNLUHI
A third light would not hurt
StrongDMM_LcF21kaCBl
the order of objects here should be table - fridge - plant - table
StrongDMM_vrzSxPfiFR
image
no wallmounts adjacent to stairs (think of how it'd look irl), no stacked wallmounts
StrongDMM_eNe0sSuPIY
the DOCKS are GOING TO STACK
The mobile dock should be facing in towards the ship anyways. I recommend putting it up front. You can just set the dheight var on it to the distance to the edge, and the template will be happy.
image
Now I know the template says /lethal but it doesn't work right now. I'd just use the normal one.
dreamseeker_cHw1sFUDAL
I would put the fax on the rear table, and toss a table wideband down in it's place.
StrongDMM_VokSkf7CT0
this layout does not work consistently. Mainly because eventually contaminants will get into the loop.
StrongDMM_gzYdQFZQ4U
Move the blast doors in a tile so they don't cover the engines. It's a style thing I've been trying to enforce for a bit
StrongDMM_3RIOKhR3Gl
using the burnt no smoking sign might be funny here

**Comment by Apogee-dev** [comment]
>fax machine

I want to leave the fax machine at the front just so it's next to all the "functional" parts of the bridge. Don't really want to have to switch that table and the one with the cigars and sundries that's all next to the vanguard's living space.

>namelist changes

No problem with doing this, I'm just curious why you want them trimmed down.

>ship name

It's a reference to this ship's design inspiration, the Cowboy-class frigate, from a Starsector mod, just like the Talos was a sideways reference to the Tarsus (and that name's also explicitly terran in origin, too, if you think about it). I don't want to get too far away from the reference here.

**Comment by Erikafox** [comment]
>No problem with doing this, I'm just curious why you want them trimmed down.

It lets the inteq namelist roll more often, which is good Solid Faction Flavoring

**Review by spockye — COMMENTED** [review]
image
make these the interior type. the tiles under the holofans start without air.

**Review by Erikafox — APPROVED** [review]

**Review by spockye — APPROVED** [review]

**Pull request merged**